client/src/cordova/plugin/apple/src/OutlinePlugin.swift

```diff
@@ -144,21 +144,24 @@ class OutlinePlugin: CDVPlugin {
     }
   }
 
-  func fetchResource(_ command: CDVInvokedUrlCommand) {
-    guard let url = command.argument(at: 0) as? String else {
-      return sendError("Missing URL", callbackId: command.callbackId)
+  func invokeMethod(_ command: CDVInvokedUrlCommand) {
+    guard let methodName = command.argument(at: 0) as? String else {
+      return sendError("Missing method name", callbackId: command.callbackId)
     }
-    DDLogInfo("Fetching resource from \(url)")
+    guard let input = command.argument(at: 1) as? String else {
+      return sendError("Missing method input", callbackId: command.callbackId)
```

Comment:
What if the method doesn't accept any input? Passing `""` or `null`?

Reply:
I'd say we should always pass an input, even if it's the empty string.